File: Source/typescript/backend/resources/MongoDbReadModelsConfiguration.ts
// Copyright (c) Dolittle. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.

import { ResourceConfiguration } from './ResourceConfiguration';

/**
 * Represents the configuration for a MongoDb connection.
 */
export class MongoDbReadModelsConfiguration extends ResourceConfiguration {
    host!: string;
    database!: string;
    useSSL!: boolean;
}

File: Source/DotNET/Backend/Resources/MongoDbReadModelsConfiguration.cs
// Copyright (c) Dolittle. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.

namespace Dolittle.Vanir.Backend.Resources
{
    public class MongoDbReadModelsConfiguration : ResourceConfiguration
    {
        public string Host { get; set; }
        public string Database { get; set; }
        public bool UseSSL { get; set; }
    }
}
